.gitignore

```diff
@@ -8,3 +8,6 @@ reports/
 
 # Node/npm
 node_modules/
+
+# VS Code Settings
+.vscode
```

Comment:
Don't add personal editor files to the repo, using `.gitignore_global`

Reply:
got it... I removed this and fixed the commit checkers @fubuloubu